lib/Spawn.js

```diff
@@ -3,7 +3,16 @@ var spawn = require('child_process').spawn;
 function Spawn(stream, command, args, name, output) {
   var exited = false;
   var ended = false;
-  var spawnedProcess = spawn(command, args);
```

Comment:
Please keep `spawnedProcess` initialized at the top of the method here (keep undefined):
```javascript
var spawnedProcess;
```